### ISSUE TITLE
Long web URL goes out of text box

### ISSUE THREAD

**Issue by bernardbaker** [thepracticaldev/dev.to#7170]
**Describe the bug**
I was viewing my notifications. And then I scrolled down to a reply. There was a URL in the box and the text bled out of the box and out of the viewport. I was using a mobile phone.

**To Reproduce**

Click the bell on mobile.
Find a notification which has a long URL in it.
See if it bleeds out of the text box.

**Expected behavior**

The word should wrap around to the next line.

**Screenshots**

![Screenshot_20200408-235842](https://user-images.githubusercontent.com/13556172/78841724-06ba2680-79f6-11ea-82f0-40f8c71d136f.png)

**Desktop (please complete the following information):**

- OS: 
- Browser: 
- Version: 

**Smartphone (please complete the following information):**

- Device: Pixel 3 XL
- OS: ANDROID
- Browser: CHROME
- Version: ??

**Additional context**

**Comment by citizen428** [comment]
Thanks for the issue @bernardbaker! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by pedrowindisch** [comment]
This problem also happens on the desktop. Working around the website, I found two solutions to this issue. 

We can fix it by applying a `word-break: break-word` property on the text. It's a quick and easy solution, but if the URL is too big, the notification can get really large. Take a look at the screenshot below.

![dev to link](https://user-images.githubusercontent.com/30203228/78855398-2eab8900-79fa-11ea-8bc0-3b2b4840c720.png)

The second solution I found requires two properties, `overflow: hidden` and `text-overflow: ellipsis`. This is how it looks like:

![dev to link-2](https://user-images.githubusercontent.com/30203228/78855840-6404a680-79fb-11ea-8183-943e59018fc6.png)

Both solutions fix the problem on all platforms and are [widely supported](https://caniuse.com/#search=text-overflow). I find the second fix better and I'd be more than happy to implement it to the codebase.

**Comment by rhymes** [comment]
I think this might be related, I just noticed it this morning

![Screen Shot 2020-04-09 at 11 23 17](https://user-images.githubusercontent.com/146201/78882401-4d406d00-7a58-11ea-98c9-b61b7bd67f41.png)

**Comment by citizen428** [comment]
@pedrowindisch Thanks 😃 If you're going to submit a PR please assign yourself the issue so we don't duplicate any efforts.

**Comment by catalinpit** [comment]
If this is not solved, and up for grabs, I'd like to have a go at it.

**Comment by Ridhwana** [comment]
@pedrowindisch just double checking if you'll be tackling this issue 😊, else we can assign it @catalinpit. Let us know:)

**Comment by pedrowindisch** [comment]
@Ridhwana I'm so sorry but I won't be able to work on this issue. I should have told y'all before. Thanks so much.

**Comment by Ridhwana** [comment]
No problem at all, thanks for letting us know. @catalinpit I've assigned the issue to you :)Feel free to ping us if you have any questions.

**Comment by catalinpit** [comment]
So which one is the recommended solution? I've seen that there was a discussion about it. 

overflow: hidden and text-overflow: ellipsis?

**Comment by ludwiczakpawel** [comment]
This probably depends on scenario.. If too long string is within the reaction notification ("_...reacted to..._") then it should be fairly easy to apply `text-overflow: ellipsis`. But for other scenarios, for example when it's within comment notification, you won't be able to do that because it doesn't have specified width and we actually can't specify width there (it's an inline element) and we need that to make ellipsis work...

**Comment by catalinpit** [comment]
Do you have any suggestions? :-)

**Comment by ludwiczakpawel** [comment]
for the latter? just go with `overflow:hidden` without any ellipsis.. it wouldn't look awesome but it would work.

**Comment by catalinpit** [comment]
Was this solved?

[img](https://i.imgur.com/yMw5p6n.png)

**Comment by Ridhwana** [comment]
@catalinpit after some digging it looks like it may have been fixed as a byproduct of [another PR](https://github.com/thepracticaldev/dev.to/pull/3262). My sincere apologies for that, I'll close this issue. Feel free to pick up another one that is not assigned if you like.
